flac_test.go

```diff
@@ -60,9 +57,11 @@ func TestSeek(t *testing.T) {
 
 	for i, pos := range testPos {
 		t.Run(fmt.Sprintf("%02d", i), func(t *testing.T) {
-			p, err := stream.Seek(pos.seek, pos.whence)
+			p, err := stream.Seek(pos.seek)
 			if err != nil {
-				t.Fatal(err)
+				if err.Error() != pos.err {
+					t.Fatal(err)
```

Comment:
No need to check expected position on error.

```suggestion
					t.Fatal(err)
					continue
```